core/src/executor/evaluate/evaluated.rs

```diff
@@ -252,6 +252,17 @@ impl<'a> Evaluated<'a> {
         .map(Evaluated::from)
     }
 
+    pub fn unary_bit_not(&self) -> Result<Evaluated<'a>> {
+        match self {
+            Evaluated::Literal(v) => Value::try_from(v).and_then(|v| v.unary_bit_not()),
+            Evaluated::Value(v) => v.unary_bit_not(),
+            Evaluated::StrSlice { source, range } => {
+                Err(EvaluateError::UnsupportedUnaryBitNot(source[range.clone()].to_owned()).into())
+            }
```

Comment:
Can we also add test code for this case?
https://coveralls.io/builds/61510412/source?filename=core%2Fsrc%2Fexecutor%2Fevaluate%2Fevaluated.rs#L259

Reply:
Let me try.
Thank you for the review.

Reply:
I pushed a new patch to test EvaluateError::UnsupportedUnaryBitNot error.
I am not sure why it cannot tirgger coverage test.